airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -354,8 +355,8 @@ def sync(self) -> None:
         self.kube_scheduler.sync()
 
         last_resource_version: dict[str, str] = defaultdict(lambda: "0")
-        while True:
-            try:
+        with contextlib.suppress(Empty):
```

Comment:
Will it break when `Empty` is raised as it used to be?

Reply:
`Empty` should be raised by `queue.get_nowait()`, so previously this would be caught by the exception block to break the loop. Now `Empty` would be raised and break out of the loop, and then suppressed by this context manager. The end result should be the same.